mithril-aggregator/src/http_server/routes/root_routes.rs

```diff
@@ -113,7 +136,7 @@ mod tests {
             .allowed_discriminants = BTreeSet::from([
             SignedEntityTypeDiscriminants::MithrilStakeDistribution,
             SignedEntityTypeDiscriminants::CardanoImmutableFilesFull,
-            SignedEntityTypeDiscriminants::CardanoTransactions,
+            SignedEntityTypeDiscriminants::CardanoStakeDistribution,
```

Comment:
Is there any specific  reason why this has changed?

Reply:
Yes, because we want to build the response with `cardano_transactions_prover: None` to avoid adding this specific data related to the `SignedEntityTypeDiscriminants::CardanoTransactions` capability.